debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/wal2json/NonStreamingWal2JsonMessageDecoder.java

```diff
@@ -92,7 +92,8 @@ public ChainedLogicalStreamBuilder optionsWithoutMetadata(ChainedLogicalStreamBu
 
     @Override
     public ChainedLogicalStreamBuilder tryOnceOptions(ChainedLogicalStreamBuilder builder) {
-        return builder.withSlotOption("include-unchanged-toast", 0);
+        // return builder.withSlotOption("include-unchanged-toast", 0);
```

Comment:
Since it's a deprecated field, I don't think we'll be adding it back.
```suggestion
```

Reply:
:+1: